src/View/Agenda/index.html.twig

```diff
@@ -1,28 +1,75 @@
 {% extends 'layout.html.twig' %}
-{% set active_page = "/concerts" %}
+{% set active_page = "/agenda" %}
+{% block title %}Agenda
+{% endblock %}
 
 {% block content %}
-	<main class="container">
-		<div class="row">
-			<h2>Agenda ({{ favorites|length }})</h2>
-		</div>
-		{% for favorite in favorites %}
-			<h4 class="card-text lead">
-				{{ favorite.date|format_datetime(locale='fr', pattern="MMMM")|capitalize }}
-			</h4>
-			<div class="card mb-3" style="max-width: 540px;">
-				<div class="row g-0">
+	<div class="row">
+		<h2>Agenda</h2>
+	</div>
+
+	{% if user and user.is_admin == false %}
+
+	{% for favorite in favorites %}
+	<div class="mx-4">
+		<h3>
+			{{ favorite.date|format_datetime(locale='fr', pattern="MMMM yyyy")|capitalize }}
+		</h3>
+	</div>
+		<div class="row justify-content-center">
+			<div class="col-12 col-md-9 col-lg-7 col-xl-6 col-xxl-5">
+				<div onclick="toggle()" class="card mb-5 card-cover overflow-hidden border-0 text-bg-white rounded-4 shadow-lg d-flex flex-row shadow-lg">
```

Comment:
No you should use an `addEventListener()` to handle the click and not the onclick attribute

Reply:
i have delete onclick